### PR TITLE
Added yon's json suggestion with slight changes

### PR DESCRIPTION
**Pull request by itayadler**
match_snapshop.json
- Added clicks
- Removed redundant corp hand and moved to HQ
- Fixed the JSON to be valid
- I think each array of cards should contain just their IDs.
- Changed archives to match the structure of the other centrals.
- Moved central servers to their own key.
- Renamed servers to remoteServers.

Also added netrunnerdb cards.json and extracted sample cards

### CODE REVIEW

File: samples/match_snapshot.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        "upgrades": [],
+        "cards": [],
+        "ice": []
+      },
+      "archives": {
+        "upgrades": [],
+        "cards": [],
+        "ice": []
+      },
+      "RND": {
```
Comment (ShayDavidson) [inline samples/match_snapshot.json:36]: R&D

File: samples/match_snapshot.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        "upgrades": [],
+        "ice": [],
+        "cards": []
+      }
+    },
+    "remoteServers": [
+      {
+       "cards": [],
+       "ice": []
```
Comment (ShayDavidson) [inline samples/match_snapshot.json:45]: all cards representations in a snapshot should be according to their `code` IMO.

File: samples/match_snapshot.json (hunk 1 above)
Comment (itayadler) [inline samples/match_snapshot.json:36]: Then u won't be able to do centralServerd.RND..

File: samples/match_snapshot.json (hunk 2 above)
Comment (itayadler) [inline samples/match_snapshot.json:45]: Ya I agree.. But its now missing the state of the card

File: samples/match_snapshot.json (hunk 1 above)
Comment (ShayDavidson) [inline samples/match_snapshot.json:36]: right :(

File: samples/match_snapshot.json (hunk 2 above)
Comment (ShayDavidson) [inline samples/match_snapshot.json:45]: so how about (with null values not having a key).

``` coffeescript
"cards": [
  {id: "02010", 
   faceDown: true,
   tokens: {
    powerTokens: 0
    virusTokens: 0
    agendaTokens: 0
    advancmentTokens: 0
   }
   recurringCreditsLeft: 0
   hostedCards: [ ... ]
 }
]
```

Comment (yonbergman) [inline samples/match_snapshot.json:45]: Why complicate things? Just support having 2 kinds of tokens (virus = red, other = blue)

``` js
{
     id: 02010
     rezzed: true/false
     sideways: true/false // optional (for UI)
     highlighted: true/false // optional (for UI)
     hostedCards: [c..]
     credits: 0
     virusTokens: 0
     advancementTokens: 0
     counters: 0 // Blue tokens (power counters / agenda counters)     
}
```

Comment (ShayDavidson) [inline samples/match_snapshot.json:45]: `rezzed` should be `faceUp` or `faceDown`, since it should also be a property for the runner.

**Comment by ShayDavidson** [comment]
We don't need this branch anymore right?

**Pull request closed without merging**